docs/src/supported-platforms.md

```diff
@@ -2,8 +2,8 @@
 
 Common requirements for all systems are:
 * minimum 16MB of RAM
-* systemd (for productive systems)
-* mosquitto minimum version 1.6 (for security reasons we recommend the latest 1.x version)
+* systemd (for production systems)
+* mosquitto minimum version 1.6 (for security reasons we recommend the latest 2.x version)
```

Comment:
Did we change the recommended mosquitto version to 2.x? I didn't know that :P

Reply:
Good point, we didn't but the parenthesis recommendation kind didn't make much to me...